README.md

```diff
@@ -1,168 +1,222 @@
 # DSP-UI library
 
-This is the demo and developing environment for DSP-UI library (@knora/ui) composed of 4 modules.
+This is the demo and developing environment for DSP-UI-LIB library (@dasch-swiss/dsp-ui) composed of 4 modules.
 
-The modules helps to create a graphical user interface, a web application to use [DSP-API](https://www.knora.org) in a quick and simple way. The modules are written in typescript to use them with [Angular](https://angular.io) (version 8). We decided to style the components and directives with [Angular Material design](https://material.angular.io).
+The modules help to create a graphical user interface, a web application to use [DSP-API](https://www.knora.org) in a quick and simple way. The modules are written in Typescript to use them with [Angular](https://angular.io) (version 9). We decided to style components and directives with [Angular Material design](https://material.angular.io).
```

Comment:
- "The modules help to create" -> "help create" (@mdelez is the "to" needed?)
- "a graphical user interface, a web application to use" -> is something missing here?
- Typescript -> TypeScript (in fact, the module is an Angular library)

Reply:
could we make separate document where we list our dependencies and versions (DSP dependencies excluded)? Otherwise this is likely to be out of date, e.g., after an Angular update.

Reply:
and for the DSP versions, we could just link to the matrix: https://docs.google.com/spreadsheets/d/1UWDFQ-2FYJw6SsKl2zznBldTPcl7Vvn3Z9ZbJfFz-UA/edit?pli=1#gid=0 Maybe the link could be made nicer.

Reply:
> could we make separate document where we list our dependencies and versions (DSP dependencies excluded)? Otherwise this is likely to be out of date, e.g., after an Angular update.

Yes, we discussed it last week. But for the release, we do not have this separate file yet. 

> and for the DSP versions, we could just link to the matrix:

I agree to link to a file like this, but it has to be public.

Reply:
> "The modules help to create" -> "help create"

DeepL suggest to write it with "to".

Reply:
> "a graphical user interface, a web application to use" -> is something missing here?

This is the whole sentence: "The modules help to create a graphical user interface, a web application to use DSP-API in a quick and simple way."
What do you think is missing? Maybe it's a little "bumpy".

Reply:
> I agree to link to a file like this, but it has to be public.

good point, @subotic Can we make the DSP version matrix public with a nice link?